stdlib/_operator.pyi

```diff
@@ -64,8 +64,8 @@ def sub(__a: Any, __b: Any) -> Any: ...
 def truediv(__a: Any, __b: Any) -> Any: ...
 def xor(__a: Any, __b: Any) -> Any: ...
 def concat(__a: Sequence[_T], __b: Sequence[_T]) -> Sequence[_T]: ...
-def contains(__a: Container[Any], __b: Any) -> bool: ...
-def countOf(__a: Iterable[Any], __b: Any) -> int: ...
+def contains(__a: Container[object], __b: object) -> bool: ...
+def countOf(__a: Iterable[object], __b: object) -> int: ...
```

Comment:
Could we make this be Iterable[_T] and _T? It might lead to false positives though.

Reply:
Do we want to? I feel like `1 in ['foo', 'bar']` is perfectly type-safe -- it's never going to raise an error, it's just always going to return `False`!

Reply:
Do we want to make it `Iterable[_T]` and `_T`? I feel like `countOf('foo', 5)` is perfectly type-safe -- it just always returns 0, no?

Reply:
Same with `1 in ['spam', 'eggs', 'bacon']` -- it seems to me that that's perfectly type-safe, it just always returns `False`

Reply:
It's safe but also useless, so we'd probably do people a service by making the type checker detect it.

However, this sort of thing also tends to cause inference issues with mypy if the types aren't exactly the same, so changing this may not work well.

Reply:
The way I see it, this is the same situation as with `list.__contains__`: the best typeshed can do is `object`, and type checkers can special-case it to show a warning when the types don't overlap.

Reply:
I agree with @Akuli -- I'm not a massive fan of having type-checkers raise an _error_ for something like this. If they want to special-case it and raise a _warning_, that's their call 🙂